src/functional/ffront/decorator.py

```diff
@@ -94,6 +94,30 @@ def _filter_closure_vars_by_type(closure_vars: dict[str, Any], *types: type) ->
     return {name: value for name, value in closure_vars.items() if isinstance(value, types)}
 
 
+def _canonicalize_args(
+    node_params: list[ct.SymbolType], args: tuple[ct.SymbolType], kwargs: dict[str, ct.SymbolType]
```

Comment:
The `list[ct.SymbolType]` annotation is not correct I believe.

Reply:
it is

Reply:
it is